gazebo_ros/CMakeLists.txt

```diff
@@ -40,6 +40,15 @@ ament_target_dependencies(gazebo_ros_init
 )
 ament_export_libraries(gazebo_ros_init)
 
+add_library(gazebo_ros_utils SHARED
+  src/utils.cpp
+)
```

Comment:
I like having a separate library, what do you think of moving conversions here?

Reply:
conversions is header only atm (can't add to library), but we could include it from utils.hpp so users only have to include one header

Reply:
Oh nice catch! I think we can leave things as they are for now then :ok_hand: